backend/tests/test_users.py

```
@@ -1,4 +1,4 @@
-from routers.users import get_db, get_current_user
+from app.routers.users import get_db, get_current_user
 from fastapi import status
 from .utils import *
```

Ruff (0.15.11)

[error] `from .utils import *` used; unable to detect undefined names

(F403)